blog/2019-03-29-experience-summary.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Haskell - An Experience Summary
+author: Sai Subramanian
```

Comment:
Have you checked if subra would like to have his github account linked too?

Reply:
@arbus

Reply:
I bugged him on a call - you can go ahead and add the handle! :)

Reply:
😂 😂 Thanks!! Will do

Reply:
Updated